### DISCUSSION TITLE
[Additional Data]how can I search by Additional Data

### DISCUSSION THREAD

**Discussion by Chris0121** [getsentry/support#50] [Support] [answered]
I have one item in Additional Data. How can I search by it?
For example : I need search the value of "graphQLOp" in Additional Data is XXX.
Thank you.
<img width="207" alt="image" src="https://user-images.githubusercontent.com/71202262/180164094-97a85c83-e5e5-4e2d-b079-d52e9396b9ba.png">

**Comment by chadwhitacre** [comment]
Is this the same ask as [this old Jira ticket](https://getsentry.atlassian.net/browse/FEEDBACK-570), @lobsterkatie?

**Comment by AbhiPrasad** [comment, marked as answer]
Extra data, displayed as "additional data" on the UI is not searchable. [Usage of additional data is deprecated](https://docs.sentry.io/platforms/javascript/enriching-events/context/#additional-data
). We recommend using context instead: https://docs.sentry.io/platforms/javascript/enriching-events/context

Context is not searchable though. As per our docs:

> If you need to be able to search on custom data, [use tags](https://docs.sentry.io/platforms/javascript/enriching-events/tags) instead.
